### PR TITLE
Expose branch to runtime

### PR DESCRIPTION
**Pull request by SferaDev**
Closes: https://github.com/xataio/client-ts/issues/519

### CODE REVIEW

File: packages/client/src/schema/repository.ts [hunk 1]
```diff
@@ -348,7 +348,7 @@ export class RestRepository<Data extends BaseData, Record extends XataRecord = D
 {
   #table: string;
   #getFetchProps: () => Promise<FetcherExtraProps>;
-  db: SchemaPluginResult<any>;
+  #db: SchemaPluginResult<any>;
```
Comment (gimenete) [inline packages/client/src/schema/repository.ts:351]: 🤔 Is this change required for the new `getConfig()` method or is this something you wanted to do anyway?

Should we maek this readonly somehow?

**Review by gimenete — APPROVED** [review]
LGTM. Just one question.

Welcome back!! 😄

File: packages/client/src/schema/repository.ts (hunk 1 above)
Comment (SferaDev) [reply packages/client/src/schema/repository.ts:351]: Not related, but I noticed it while doing this change.

We don't have the db object in the interface definition. So was available because of JS even if not typed in TS. We shouldn't expose the internals and only public facing API.
